plugins/git/internal-functions

```diff
@@ -112,10 +112,12 @@ cmd-git-auth() {
   if [[ -z "$USERNAME" ]]; then
     dokku_log_info1 "Removing netrc auth entry for host $HOST"
     netrc unset "$HOST"
+    rm "${DOKKU_ROOT}/.netrc"
```

Comment:
We probably don't need the remove?

Reply:
Yes, I guess it is fine to leave the .netrc file once it has been created.